dcpy/connectors/edm/publishing.py

```diff
@@ -401,7 +401,7 @@ def _cli_wrapper_upload(
     acl_literal = s3.string_as_acl(acl)
     if not output_path.exists():
         raise Exception(f"Path {output_path} does not exist")
-    build_name = build or os.environ.get("BUILD_NAME")
+    build_name = build or os.environ["BUILD_NAME"]
     if build_name is None:
         raise Exception(
```

Comment:
✂️  This will never get raised now

Reply:
see [0a04fdd](https://github.com/NYCPlanning/data-engineering/pull/510/commits/0a04fdd846212106e953bc85d5c5503f4d02103f). still seemed worth raising if `BUILD_NAME = ""` since nothing downstream of this seems to